src/screen.ts

```diff
@@ -37,15 +44,22 @@ const logTestingPlaygroundURL = (element = getDocument().body) => {
   )
 }
 
-const initialValue = {debug, logTestingPlaygroundURL}
+const initialValue: {
+  [key in keyof typeof queries | 'debug' | 'logTestingPlaygroundURL']?: Function
+} = {debug, logTestingPlaygroundURL}
```

Comment:
Does 
```suggestion
const initialValue = {debug, logTestingPlaygroundURL} as const
```
work as well?